static/app/views/projectInstall/issueAlertNotificationOptions.tsx

```diff
@@ -0,0 +1,277 @@
+import {useCallback, useEffect, useMemo, useState} from 'react';
+import styled from '@emotion/styled';
+
+import MultipleCheckbox from 'sentry/components/forms/controls/multipleCheckbox';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {IssueAlertActionType} from 'sentry/types/alerts';
+import type {OrganizationIntegration} from 'sentry/types/integrations';
+import {useApiQuery} from 'sentry/utils/queryClient';
+import useApi from 'sentry/utils/useApi';
+import useOrganization from 'sentry/utils/useOrganization';
+import SetupMessagingIntegrationButton, {
+  MessagingIntegrationAnalyticsView,
+} from 'sentry/views/alerts/rules/issue/setupMessagingIntegrationButton';
+import MessagingIntegrationAlertRule from 'sentry/views/projectInstall/messagingIntegrationAlertRule';
+
+export const providerDetails = {
+  slack: {
+    name: t('Slack'),
+    action: IssueAlertActionType.SLACK,
+    placeholder: t('channel, e.g. #critical'),
+    makeSentence: ({providerName, integrationName, target}) =>
+      tct(
+        'Send [providerName] notification to the [integrationName] workspace to [target]',
+        {
+          providerName,
+          integrationName,
+          target,
+        }
+      ),
+  },
+  discord: {
+    name: 'Discord',
+    action: IssueAlertActionType.DISCORD,
+    placeholder: 'channel ID or URL',
+    makeSentence: ({providerName, integrationName, target}) =>
+      tct(
+        'Send [providerName] notification to the [integrationName] server in the channel [target]',
+        {
+          providerName,
+          integrationName,
+          target,
+        }
+      ),
+  },
+  msteams: {
+    name: 'MSTeams',
```

Comment:
Is it called `MSTeams` or just `Teams`? Was pretty sure this product is just called Teams

We really would want a translator note here, but as mentioned before, we don't support that so doing this might end up being confusing for our translators :(

```suggestion
    name: t('Teams'),
```

maybe better is actually

```suggestion
    name: t('MS Teams'),
```

Reply:
I believe it's called `MS Teams`, or at least that's what we have everywhere else in our codebase